

class UsingError(Exception):
    pass


class UsingSourceError(Exception):
    pass


class UsingDestinationError(Exception):
    pass
